code/game/area/magmoor_digsite.dm

```diff
@@ -0,0 +1,363 @@
+// Magmoor Digsite IV - Hathkar
+
+//Base Area
+
+/area/magmoor
```

Comment:
might want to also define `outside`.
Its coming with weather, for now you can just add `var/outside = TRUE/FALSE` here and remove `var/` when it gets merged.

Reply:
Done